velox/type/Variant.cpp

```diff
@@ -212,11 +217,16 @@ std::string variant::toJson(const TypePtr& type) const {
       std::string b{};
       b += "[";
       bool first = true;
+      uint32_t idx = 0;
+      VELOX_USER_CHECK_EQ(
+          row.size(),
+          type->size(),
+          "variant children count does not match the input type children count");
```

Comment:
nit: perhaps, "Wrong number of fields in a struct in variant::toJson"

Reply:
Fixed.